changelog.txt

```diff
@@ -1,3 +1,6 @@
+0.84.1
+ * Fixed a bug simultaneous opening and closing Realm may produce inconsistent Realm instance and it gets NPE on column access. See #1646 for details.
```

Comment:
I would rewrite it to this:

`Fixed a bug where simultaneous opening and closing a Realm on different threads might result in a NullPointerException (#1646).`